requirements.txt

```diff
@@ -2,7 +2,7 @@ aenum
 pydantic
 pyyaml==6.0.1
 sympy==1.12
-numpy==1.23.*
+numpy==1.26.4
```

Comment:
appabuild needs a 1.23.* numpy version, and requires apparun. Can we safely update numpy on both repo and keep a compatiblity with python 3.9 ?

Reply:
Python>=3.10 required, seems acceptable but documentation needs to be updated